### ISSUE TITLE
How add field to user table

### ISSUE THREAD

**Issue by ekoorosh** [thedevdojo/voyager#1392]
- Laravel Version: 5.4.23
- Voyager Version: 0.11.14
- PHP Version:5.6.25
- Database Driver & Version:mysql & 5.7.14

Hello 
I add a field to user table but when I go to users and edit a user the field doesn't show for edit. what do I do?

**Comment by fletch3555** [comment]
Users has [custom views](https://github.com/the-control-group/voyager/tree/master/resources/views/users).  You will need to override these views to see the new field.

**Comment by owldesign** [comment]
I learned that even if you add database field to Users table and override `edit-add.blade.php` view. It will not save those custom fields.

Reason because those fields aren't part of the `data_rows` table yet. How I went about it is:
1. Go edit `users` BREAD
2. Just drag any of the fields around, and save.. (there is a dragger icon next to each field row)

I guess that retriggers something in the Voyager and it updates the `data_rows` table with the new User table fields.
